### PR TITLE
Added capability to fetch issues with configurable tags.

### PR DESCRIPTION
**Pull request by arnabkaycee**
Added capability to fetch issues from hyperledger orgs with configurable tags.
Change config.yaml
```yaml
issue-tags:
  - "good first issue"
  - "help wanted"
```
to add the tags you need.
Generated file will be `issue-summary.html`
This file/template needs to be further designed for aesthetics.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -3,7 +3,13 @@ organizations:
...
+# add the tags to be generated
+issue-tags:
+  - "good first issue"
+  - "help wanted"
 days: 7
+issue-created-history-days: 100
 filename: "data.json"
 pr-summary-filename: "pr-summary.html"
 release-summary-filename: "release-summary.html"
+issue-summary-filename: "issue-summary.html"
```
Comment (arsulegai) [inline config.yaml]: [nitpick] Please add a new-line at the EOF.

File: config.yaml [hunk 2]
```diff
@@ -3,7 +3,13 @@ organizations:
       name: "hyperledger"
   - organization:
       name: "hyperledger-labs"
+# add the tags to be generated
+issue-tags:
+  - "good first issue"
+  - "help wanted"
 days: 7
+issue-created-history-days: 100
```
Comment (arsulegai) [inline config.yaml:11]: [nitpick] Could be done as a next PR as well. Suggestion: Can the list of information to be pulled be kept as an array, each having their own timer value (time could be in minutes or seconds as well)?

File: github.go [hunk 3]
```diff
@@ -222,3 +234,69 @@ func (c Client) ListReleases(org string, repos []string, daysCount int) ([]Relea
...
+func (c Client) IssueWithLabels(org string, repos []string, issueLabels []string, daysCount int) ([]IssueList, error) {
+	var issueList []IssueList
+
+	dayDiff := daysCount * -1
+
+	//get open issues to be worked on and which has not been assigned to someone
+	issueListOptions := &github.IssueListByRepoOptions{
+		State:    "open",
+		Assignee: "",
+		Labels:   issueLabels,
```
Comment (arsulegai) [inline github.go]: Q: Query has to fetch both `good first issue` and `help wanted`. i.e. Even if the issue is not tagged as `good first issue`, all `help wanted` can be listed in the website. Will this query statement give that result?

File: github.go [hunk 4]
```diff
@@ -222,3 +234,69 @@ func (c Client) ListReleases(org string, repos []string, daysCount int) ([]Relea
...
+			if response.StatusCode != http.StatusOK {
+				return nil, errors.New("Could not get the response for fetching issues")
+			}
+			for _, issue := range issues {
+
+				publishedDate := issue.GetCreatedAt()
+				startDate := time.Now().AddDate(0, 0, dayDiff)
+				log.Println("publishedDate", publishedDate, "start date", startDate, " if condition", publishedDate.Before(startDate))
+
+				if publishedDate.Before(startDate) {
```
Comment (arsulegai) [inline github.go:276]: Q: The assumption with PR list was that it was listed in descending order of creation time. Does that hold true with this API as well?

File: issues.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
+body {
+    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```
Comment (arsulegai) [inline issues.css]: Can this file be placed in a static folder?
[nitpick] This repository has to do better job at code organization. It needs cleaning and reorganizing in folder structure.

File: main.css [hunk 6]
```diff
@@ -57,7 +57,7 @@ ol li::before {
     height: 30px;
```
Comment (arsulegai) [inline main.css]: Why is this file move out of `static` folder?

File: static/issue-template.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+                    {{end}}
+                </ol>
+                {{end}}
+            </ol>
+            {{end}}
+        </ol>
+    </div>
+</body>
+
+</html>
```
Comment (arsulegai) [inline static/issue-template.html]: [nitpick] New line character at the EOF

File: issues.css (hunk 5 above)
Comment (arnabkaycee) [reply issues.css]: The generated html cannot reference the css file within the static folder. We can restructure it like this
```bash
project-root
  - html
    - css
    - template
    - generated
```

File: config.yaml (hunk 2 above)
Comment (arnabkaycee) [reply config.yaml:11]: Sure I will structure this better.

File: issues.css (hunk 5 above)
Comment (arsulegai) [reply issues.css]: Sounds like a good idea.

File: github.go (hunk 3 above)
Comment (arnabkaycee) [reply github.go]: Thanks for this. Turns out this API uses an AND clause to search issues with all matching tags. I had to manually add that filter. Now it fetches all issues and adds them to the output if at least one of the tags match.

File: github.go (hunk 4 above)
Comment (arnabkaycee) [reply github.go:276]: Added this search criteria explicitly

File: main.css (hunk 6 above)
Comment (arnabkaycee) [reply main.css]: Moved back in `html` folder with proper folder structure

**Comment by arsulegai** [comment]
Thanks for the PR, I will merge it at current state. Restructuring can be done as an improvement in upcoming PRs.

**Pull request merged**